src/utilities.ts

```diff
@@ -73,15 +82,15 @@ function totalLineOne(orderItem, currencySymbol) {
   }
 }
 
-function totalLineTwo(orderItem, currencySymbol) {
+export function totalLineTwo(orderItem: OrderItem, currencySymbol?: string) {
   if (orderItem.purchasableType === 'bundle' && !orderItem.gift && orderItem.coupon && orderItem.coupon.duration !== 'forever') {
-    return priceFormat(totalRecurring(orderItem), currencySymbol) + ' / ' + orderItem.interval;
+    return priceFormat(totalRecurring(orderItem) || 0, currencySymbol) + ' / ' + orderItem.interval;
```

Comment:
utility `totalRecurring` returns a `number | null` type. add fallback to `0` here to account for `null` returns.